gamemode/modules/commands/sv_commands.lua

```diff
@@ -246,10 +252,32 @@ end
 
 net.Receive( "RVR_Commands_runConsoleCommand", onRunConsoleCommand )
 
-commands.register( "help", { "command" }, { "string" }, RVR_USER_ALL, function( ply, command )
-    if not commands.commands[command] then
-        return "Help: Command \"" .. command .. "\" does not exist"
-    end
+local function initializeBaseCommands()
+    commands.register( "usage", { "command" }, { "string" }, RVR_USER_ALL, function( ply, command )
+        if not commands.commands[command] then
+            return "Help: Command \"" .. command .. "\" does not exist"
+        end
+
+        return commands.commands[command].description
+    end, "Prints the usage and description of a command" )
+
+    commands.register( "help", {}, {}, RVR_USER_ALL, function( ply )
+        local plyUserGroup = RVR.getUserGroup( ply )
+
+        ply:PrintMessage( HUD_PRINTCONSOLE, "----- RaftVRaft Commands -----" )
+
+        for commandName, commandData in pairs( commands.commands ) do
+            if plyUserGroup < commandData.userGroup then continue end
+
+            local description = commandName .. ":\n" ..  commandData.description .. "\n "
+
+            ply:PrintMessage( HUD_PRINTCONSOLE, description )
```

Comment:
```suggestion
            local shouldPrint = plyUserGroup < commandData.userGroup

            local description = commandName .. ":\n" ..  commandData.description .. "\n "

            if shouldPrint then ply:PrintMessage( HUD_PRINTCONSOLE, description ) end
```

Reply:
Waste of computational power and the end line is too long

Reply:
You really gonna come out here and compute the description potentially unnecessarily?
Are you actually insane?
I'm not sure I can tollerate this kind of behaviour in this repo.
Please take a long hard think about your actions.

I'll be talking to your manager.

Reply:
I can almost guarantee the extra computation wont be an issue unless you are literally using  a potato for a computer

the line for the description is also longer

Reply:
you're hiding the extra scope by making it single line, but that extra scope still exists
and it'll haunt you in your dreams